evalml/tests/component_tests/test_components.py

```diff
@@ -521,6 +524,10 @@ def test_transformer_transform_output_type(X_y_binary):
             if isinstance(component, SelectColumns):
                 assert transform_output.shape == (X.shape[0], 0)
                 assert isinstance(transform_output.columns, pd.Index)
+            elif isinstance(component, PCA):
+                assert transform_output.shape[0] == X.shape[0]
+                assert transform_output.shape[1] < X.shape[1]
```

Comment:
Oh cool, I don't know the algorithm but we'll always end up with less columns? There's never the case where `transform_output.shape[1] = X.shape[1]`? :o

Reply:
Ah, good point! It is possible, since the default is to keep the features that represent *at least* 95% of the variance, which could end up being all features in some cases. I'll update this tot be `<=` instead.